assignments/array-methods.js

```diff
@@ -56,21 +56,34 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(runner => {fullName.push(`${runner.first_name} ${runner.last_name}`)});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+allCaps = runners.map( runner => `${runner.first_name.toUpperCase()}`);
+
+
+
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+
+largeShirts = runners.filter( runner => runner.shirt_size === "L" ? runner : null );
```

Comment:
excellent use of a ternary operator here! you can shorten this even further because `filter` has an implicit `null` for all items that don't meet the conditional statement, so this can be written simply as `runner => runner.shirt_size === "L"` and leave it at that